public/css/gogs.css

```diff
@@ -2096,6 +2173,27 @@ footer .container .links > *:first-child {
   width: 15px;
   height: 15px;
 }
+#avatar-arrow:before,
```

Comment:
Those are from my previous pr for the arrows on the comment form, gogs.css wasn't generated after you merged it